components/org.wso2.carbon.identity.authenticator.risk.analytics.endpoint/src/main/java/org/wso2/carbon/identity/authenticator/risk/analytics/endpoint/util/ServiceValueHolder.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.authenticator.risk.analytics.endpoint.util;
+
+import org.wso2.carbon.context.PrivilegedCarbonContext;
+import org.wso2.carbon.event.stream.core.EventStreamService;
+import org.wso2.carbon.event.template.manager.core.TemplateManagerService;
+import org.wso2.carbon.identity.authenticator.risk.analytics.endpoint.ResultContainer;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Holder object to hold references which are needed by multiple classes.
+ */
+public class ServiceValueHolder {
+    private static ServiceValueHolder instance;
+    private static EventStreamService eventStreamService;
+    private static TemplateManagerService templateManagerService;
+    private static Map<String, ResultContainer> resultContainerMap = new ConcurrentHashMap<>();
+
+    private ServiceValueHolder() {
+        //DO nothing
```

Comment:
Add space at the start of the comment and properly capitalize? ("// Do nothing")